sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/ecert-full-time-process-integration.scheduler.ts

```diff
@@ -3,48 +3,67 @@ import { ECertFileHandler } from "@sims/integrations/esdc-integration";
 import { QueueService } from "@sims/services/queue";
 import { QueueNames } from "@sims/utilities";
 import { Job, Queue } from "bull";
-import { QueueProcessSummary } from "../../../models/processors.models";
 import { BaseScheduler } from "../../base-scheduler";
-import { ESDCFileResult } from "../models/esdc.models";
+import { FullTimeCalculationProcess } from "@sims/integrations/services/disbursement-schedule/e-cert-calculation";
+import {
+  InjectLogger,
+  LoggerService,
+  ProcessSummary,
+} from "@sims/utilities/logger";
+import {
+  getSuccessMessageWithAttentionCheck,
+  logProcessSummaryToJobLogger,
+} from "../../../../utilities";
 
 @Processor(QueueNames.FullTimeECertIntegration)
 export class FullTimeECertProcessIntegrationScheduler extends BaseScheduler<void> {
   constructor(
     @InjectQueue(QueueNames.FullTimeECertIntegration)
     schedulerQueue: Queue<void>,
     queueService: QueueService,
+    private readonly fullTimeCalculationProcess: FullTimeCalculationProcess,
     private readonly eCertFileHandler: ECertFileHandler,
   ) {
     super(schedulerQueue, queueService);
   }
 
   /**
-   * Process Full-Time disbursements available to be sent to ESDC.
+   * Process full-Time disbursements available to be sent to ESDC.
```

Comment:
"full-time"?

Reply:
Sorry, I did not follow. Can you elaborate please?